src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java

```diff
@@ -86,11 +85,12 @@ public StreamingResponseBody streamEventsFromPartition(
                 }
 
                 // deserialize cursors
-                Optional<List<Cursor>> cursors = Optional.empty();
+                List<Cursor> cursors = null;
                 if (cursorsStr != null) {
                     try {
-                        cursors = Optional.of(jsonMapper.<List<Cursor>>readValue(cursorsStr,
-                                new TypeReference<ArrayList<Cursor>>() {}));
+                        cursors = jsonMapper.<List<Cursor>>readValue(cursorsStr,
+                                new TypeReference<ArrayList<Cursor>>() {
+                                });
                     } catch (IOException e) {
                         writeProblemResponse(response, outputStream, HttpStatus.BAD_REQUEST.value(),
```

Comment:
stream_limit has default of 0
stream_timeout has default of 0 (changed in the api now) 
batch_keep_alive_limit is 0 and was renamed to stream_keep_alive_limit